app/components/organization-profile.js

```diff
@@ -2,4 +2,21 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
   classNames: ['organization-profile'],
+
+  membership: Ember.computed('organization.organizationMemberships', 'session.currentUser', function() {
+    let user = this.get('session.currentUser');
+    if (user) {
+      let memberships = this.get('organization.organizationMemberships');
+      let membership = memberships.find(this.findMembership.bind(this));
+      return membership;
+    }
+  }),
+
+  findMembership: function(item) {
+    let itemMemberId = item.belongsTo('member').id();
+    let itemOrganizationId = item.belongsTo('organization').id();
+    let currentUserId = this.get('session.currentUser.id');
+    let organizationId = this.get('organization.id');
+    return (itemMemberId === currentUserId) && (itemOrganizationId === organizationId);
+  },
```

Comment:
@begedin At some point we're going to move to project-based permissioning, too, so things will only grow more and more complex. It will be nice to consolidate that logic. As you can see, there's a lot of repetition in my approach.

Can we have the second pass happen in the context of this PR? I don't really want to merge this as-is (even if I get the tests passing). I'd like for you to take point on it and I can work on some other things.